openslides/users/views.py

```diff
@@ -478,3 +162,58 @@ def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['user'] = self.request.user
         return kwargs
+
+
+class UserLoginView(AjaxMixin, View):
+    """
+    Login the user via ajax.
+    """
+
+    def get_ajax_context(self, **context):
+        if self.success:
+            context.update({
+                'success': True,
+                'user_id': self.user.pk})
+        else:
+            context.update({'success': False})
+
+        return super().get_ajax_context(**context)
+
+    def post(self, request, *args, **kwargs):
+        try:
+            data = json.loads(request.body.decode('utf-8'))
```

Comment:
Which is the same: https://docs.python.org/3/library/stdtypes.html#bytes.decode

but ok: done